src/calculation/DetailedResultsBuilder.ts

```diff
@@ -304,10 +304,27 @@ export function createPatientPopulationValues(
     populationGroup.stratifier.forEach(strata => {
       if (strata.criteria?.expression) {
         const value = patientResults[strata.criteria?.expression];
+
+        // if the cqfm-appliesTo extension is present, then we want to consider the result of that
+        // population in our stratifier result
+        const appliesToExtension = strata.extension?.find(
+          e => e.url === 'http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-appliesTo'
+        );
+
+        let popValue = true;
+        if (appliesToExtension) {
+          const popCode = appliesToExtension.valueCodeableConcept?.coding?.[0].code;
+          if (popCode) {
+            popValue = patientResults[popCode];
```

Comment:
This isn't the right place to get the results for population as the `patientResults` is the raw data from the engine. The define statement that provides the results might not match the code. Should look back at the `populationResults` that have already been collected and find by `populationType`.

Reply:
Additionally the populationResults will not be accurate at this point and should happen after the call to `handlePoulationValues` on line 35.